.travis.yml

```diff
@@ -21,8 +21,6 @@ matrix:
           env: SYMFONY_VERSION=^4.3
         - php: 7.3
           env: SYMFONY_VERSION=^5.0
-        - php: 7.4
-          env: SYMFONY_VERSION=^5.0
```

Comment:
Why removing this test?

Reply:
This case covered by third job: https://github.com/odolbeau/phone-number-bundle/pull/41/checks?check_run_id=1264722969

Reply:
:ok_hand: